test/OrchardCore.Benchmarks/RuleBenchmark.cs

```diff
@@ -61,16 +61,10 @@ static RuleBenchmark()
 
         [Benchmark(Baseline = true)]
 #pragma warning disable CA1822 // Mark members as static
-        public void EvaluateIsHomepageWithJavascript()
-        {
-            Convert.ToBoolean(_engine.Evaluate(_scope, "isHomepage()"));
-        }
+        public void EvaluateIsHomepageWithJavascript() => _engine.Evaluate(_scope, "isHomepage()");
```

Comment:
Does this still run properly? I think it should, but since its return value now is a discard, perhaps some compiler optimization will unwantedly get rid of it.

Reply:
I don't see a value there, you can look into the other benchmark in the same file, no conversion included